<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sub3dtoolgui</class>
 <widget class="QWidget" name="sub3dtoolgui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>577</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>sub3dtoolgui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sub3dtool-gui-images.qrc">
    <normaloff>:/icons/images/logo_sub3dtool.png</normaloff>:/icons/images/logo_sub3dtool.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabFilesSelect">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabSingleFile">
      <attribute name="title">
       <string>Single File</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Input File</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEditFileIn"/>
          </item>
          <item>
           <widget class="QPushButton" name="buttonOpenFileIn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="sub3dtool-gui-images.qrc">
              <normaloff>:/icons/images/folder-documents-icon.png</normaloff>:/icons/images/folder-documents-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Output File</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEditFileOut"/>
          </item>
          <item>
           <widget class="QPushButton" name="buttonOpenFileOut">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="sub3dtool-gui-images.qrc">
              <normaloff>:/icons/images/folder-documents-icon.png</normaloff>:/icons/images/folder-documents-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabMultipleFiles">
      <attribute name="title">
       <string>Multiple Files</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>InFile</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>OutFile</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="buttonAddMultipleFiles">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonRemoveMultpleFiles">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>3D Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QPushButton" name="button3DTB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub3dtool-gui-images.qrc">
          <normaloff>:/icons/images/logo_TB.png</normaloff>:/icons/images/logo_TB.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="button3DN3D">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub3dtool-gui-images.qrc">
          <normaloff>:/icons/images/logo_N3D.png</normaloff>:/icons/images/logo_N3D.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="button3DSBS">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="sub3dtool-gui-images.qrc">
          <normaloff>:/icons/images/logo_SBS.png</normaloff>:/icons/images/logo_SBS.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>64</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label3DSBS">
        <property name="text">
         <string>Side by Side</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label3DTB">
        <property name="text">
         <string>Top - Bottom</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label3DN3D">
        <property name="text">
         <string>No 3D</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Screen Resolution:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboResolution">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="maxVisibleItems">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Font:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFontComboBox" name="fontComboBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="fontFilters">
           <set>QFontComboBox::AllFonts</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Size:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFontSize">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAdvance">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Advance...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="buttonAbout">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>About...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonConvert">
       <property name="text">
        <string>Convert</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sub3dtool-gui-images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonExit</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpenFileIn</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>getFileIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonOpenFileOut</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>getFileOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3DN3D</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>change3DModeN3D()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3DSBS</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>change3DModeSBS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button3DTB</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>change3DModeTB()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonConvert</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>convert()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFileIn</sender>
   <signal>editingFinished()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>filesChangedIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditFileOut</sender>
   <signal>editingFinished()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>filesChangedOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboFontSize</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>setFontSize(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboFontSize</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>setFontSize(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboResolution</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>setResolution(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboResolution</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>setResolution(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>330</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontComboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>setFont(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fontComboBox</sender>
   <signal>editTextChanged(QString)</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>setFont(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAbout</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdvance</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>showAdvance()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAddMultipleFiles</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>addMultipleFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveMultpleFiles</sender>
   <signal>clicked()</signal>
   <receiver>sub3dtoolgui</receiver>
   <slot>removeMultipleFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>398</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>closeProgram()</signal>
  <slot>initGui()</slot>
  <slot>checkTool()</slot>
  <slot>getFileOut()</slot>
  <slot>getFileIn()</slot>
  <slot>change3DModeSBS()</slot>
  <slot>change3DModeTB()</slot>
  <slot>change3DModeN3D()</slot>
  <slot>convert()</slot>
  <slot>filesChangedIn()</slot>
  <slot>setFontSize(QString)</slot>
  <slot>setResolution(QString)</slot>
  <slot>setFont(QString)</slot>
  <slot>showAbout()</slot>
  <slot>exit()</slot>
  <slot>showAdvance()</slot>
  <slot>filesChangedOut()</slot>
  <slot>addMultipleFiles()</slot>
  <slot>removeMultipleFiles()</slot>
 </slots>
</ui>
